src/VirtoCommerce.LuceneSearchModule.Data/LuceneSearchProvider.cs

```diff
@@ -133,9 +133,10 @@ public virtual Task<IndexingResult> RemoveAsync(string documentType, IList<Index
                     {
                         if (!string.IsNullOrEmpty(document.Id))
                         {
+                            var mgrWriter = new TrackingIndexWriter(writer);
```

Comment:
There is no such word as mgr.
What is mgr?

Reply:
it's my fault , just copy/paste from src code